packages/react-native/React/Base/RCTUtils.h

```diff
@@ -56,6 +56,7 @@ RCT_EXTERN CGFloat RCTScreenScale(void);
 RCT_EXTERN CGFloat RCTFontSizeMultiplier(void);
 RCT_EXTERN CGSize RCTScreenSize(void);
 RCT_EXTERN CGSize RCTViewportSize(void);
+RCT_EXTERN UIDeviceOrientation RCTDeviceOrientation(void);
```

Comment:
do we really need this extra function in the header? This will be a new public API that we need to mmaintain. Given that it is used only in RCTUtils.mm, can we keep it private in the .mm file instead?

Reply:
@cipolleschi We probably don't need to expose it. I removed it.